### ISSUE TITLE
Looks like one of the buildings outside of the map could be bought? No address?

### ISSUE THREAD

**Issue by rpgforyou** [hovgaardgames/bigambitions#1234]
### Build number

1015

### Bug description

Looks like one of the buildings outside of the map could be bought? No address?

### Screenshots or videos

![afbeelding](https://user-images.githubusercontent.com/9866863/180621249-538f6158-be17-45bc-aa5b-d5fea1010b36.png)

**Comment by jokoho48** [comment]
need more specific location of the building.

**Comment by rpgforyou** [comment]
Found it! I think?
Looks at the park? At the split? On the image from real estate. When I clicked to mark it on Voogle it didn't do anything. This is the best I could do. 

![afbeelding](https://user-images.githubusercontent.com/9866863/181122225-4d7e2c4c-a7f2-4f96-8ee3-d9d1bdbd9e44.png)
![afbeelding](https://user-images.githubusercontent.com/9866863/181122161-e443aa42-d975-40f4-954a-ea8e9c707b5c.png)
![afbeelding](https://user-images.githubusercontent.com/9866863/181122208-f0f714dc-6100-4c21-890d-93eaed49129f.png)
